sources/install/package_forensic.sh

```diff
@@ -121,6 +121,13 @@ function install_chainsaw() {
     add-to-list "chainsaw,https://github.com/WithSecureLabs/chainsaw,Rapidly Search and Hunt through Windows Forensic Artefacts"
 }
 
+function install_magika() {
+    colorecho "Installing magika"
+    pip install magika
+    add-history magika
+    add-test-command "magika --help"
+    add-to-list"magika,https://github.com/google/magika.git,Magika is a novel AI powered file type detection tool that relies on the recent advance of deep learning to provide accurate detection."
+}
```

Comment:
This tool doesn't seem to have anything to do with adding lists. Can you remove it and make a second pull request if you want to add it for exegol ?